frontend/src/components/ShootWarnings/ConstraintWarningMessage.vue

```diff
@@ -0,0 +1,29 @@
+<!--
```

Comment:
Rename the the file to `ConstraintMessage.vue` or even more generic

Reply:
done